src/prefect/events/clients.py

```diff
@@ -140,6 +140,10 @@ def reset(cls) -> None:
         cls.last = None
         cls.all = []
 
+    @classmethod
+    def reset_events(self) -> None:
+        self.events = []
```

Comment:
nit: you should replace `self` with `cls` since this is a `classmethod`

Reply:
woops typo, thank you!